### PR TITLE
README.md: update and correct the FreeBSD section, and a little more

### PR DESCRIPTION
**Pull request by grahamperrin**
FreeBSD bugs: 

* [260763 – Advocacy: Sites using FreeBSD: update](https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=260763)
* [260937 – Community page update](https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=260937)
* [260968 – Support: Web resources (page update)](https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=260968)

### CODE REVIEW

**Comment by grahamperrin** [comment]
My first edition of this (before the PR) added `/f/freebsd` to the FreeBSD section. It took a while for me to realise that the subreddit is near the foot of the page (easily overlooked, despite the mention under _Structure_ near the head of the page).

I'd like to move subreddit listings; if there's an associated system, the listing should be **with** (not separate from) the system. OK?

**Comment by hovancik** [comment]
I guess that's fine :)

File: README.md [hunk 1]
```diff
@@ -155,111 +168,97 @@ pfSense® software is a free, open source customized distribution of FreeBSD spe
...
 
-* [BSDNow](http://www.bsdnow.tv/) -  weekly BSD podcast, featuring news, interviews and tutorials, in MP3, OGG and MP4
-* [BSDTalk](http://bsdtalk.blogspot.com/) - Talking about the BSD family of free operating systems.
-* [Garbage (archive)](https://garbage.jcs.org/) - A podcast about technology, much of it garbage. (Archived episodes)
+* [BSDNow](https://www.bsdnow.tv/) – weekly BSD podcast, featuring news, interviews and tutorials, in MP3, OGG and MP4
+* [BSDTalk](https://bsdtalk.blogspot.com/) – talking about the BSD family of free operating systems
+* [Garbage (archive)](https://garbage.jcs.org/) – a podcast about technology, much of it garbage (archived episodes).
 
-## Forums and Communities
+## Forums and communities
```
Comment (grahamperrin) [inline README.md:227]: Alphabetical order for the subsections below?

File: README.md [hunk 2]
```diff
@@ -155,111 +168,97 @@ pfSense® software is a free, open source customized distribution of FreeBSD spe
...
-
-### User Groups
-
-* [AMBUG](https://ambug.org/) - Armenia BSD User Group
-* [BSD-PL](https://bsd-pl.org/en) - Polish BSD User Group
-* [HELBUG](http://helbug.fi) - Helsinki area BSD User Group
+* [AMBUG](https://ambug.org/) – Armenia BSD User Group
+* [BSD-PL](https://bsd-pl.org/en) – Polish BSD User Group
+* [HELBUG](http://helbug.fi/) – Helsinki area BSD User Group
 * [FreeBSD Groups](https://www.freebsd.org/usergroups.html)
```
Comment (grahamperrin) [inline README.md:250]: _FreeBSD user groups_ is in the `www.freebsd.org` domain. Move up to the FreeBSD section?

File: README.md [hunk 3]
```diff
@@ -155,111 +168,97 @@ pfSense® software is a free, open source customized distribution of FreeBSD spe
...
 * [Knoxville BSD User Group](http://knoxbug.org/)
-* [MetaBUG](http://www.metabug.org/) - A Global BSD Users Group Federation
+* [MetaBUG](https://www.metabug.org/) – a global BSD users group federation
 * [NetBSD Groups](https://www.netbsd.org/community/groups.html)
```
Comment (grahamperrin) [inline README.md:253]: _NetBSD user groups and regional resources_ is in the `www.netbsd.org` domain. Move up to the NetBSD section?

File: README.md [hunk 4]
```diff
@@ -155,111 +168,97 @@ pfSense® software is a free, open source customized distribution of FreeBSD spe
...
 * [New York City \*BSD User Group](https://nycbug.org/)
-* [New Delhi BSD User Group](http://ndbug.in)
-* [OpenBSD Groups](http://www.openbsd.org/groups.html)
+* [New Delhi BSD User Group](https://ndbug.in/)
+* [OpenBSD user groups](https://www.openbsd.org/groups.html)
```
Comment (grahamperrin) [inline README.md:256]: _OpenBSD user groups_ is in the `www.openbsd.org` domain. Move up to the OpenBSD section?

File: README.md [hunk 5]
```diff
@@ -155,111 +168,97 @@ pfSense® software is a free, open source customized distribution of FreeBSD spe
...
+* [dmesgd](https://dmesgd.nycbug.org/) – a user-submitted repository of searchable \*BSD dmesgs, maintained by NYC\*BUG
 
 ## Learning
 
-* [The BSD Certification Group](http://www.bsdcertification.org/) - The BSD Certification Group Inc. is a non-profit organization committed to creating and maintaining a global certification standard for system administration on BSD based operating systems.
-* [TeachBSD](http://teachbsd.org/) - Teaching Operating Systems with Tracing : a set of courses for students and software practitioners. 
+* [The BSD Certification Group](http://www.bsdcertification.org/) – a non-profit organization committed to creating and maintaining a global certification standard for system administration on BSD based operating systems.
+* [TeachBSD](http://teachbsd.org/) – Teaching Operating Systems with Tracing : a set of courses for students and software practitioners. 
 
 ## Books
```
Comment (grahamperrin) [inline README.md:270]: Move the system-specific books up to the respective system sections?

**Review by hovancik — APPROVED** [review]

**Comment by grahamperrin** [comment]
Thank you! Comments above (the review) for a later date, maybe?

**Comment by hovancik** [comment]
> Thank you! Comments above (the review) for a later date, maybe? 


What do you mean?

**Comment by grahamperrin** [comment]
Maybe I'm not using the review feature properly, do you see this, above? 

image

**Comment by hovancik** [comment]
Gotcha, not seeing that. I think you need to hit "Finish review" or sth similar at the top of the page. (As there is a "Pending" at your comments)

**Comment by grahamperrin** [comment]
Yeah, _Pending_ made sense after I clicked through. Sorry!

File: README.md (hunk 2 above)
Comment (hovancik) [reply README.md:250]: 👍

File: README.md (hunk 1 above)
Comment (hovancik) [reply README.md:227]: 👍 I think it makes sense

File: README.md (hunk 5 above)
Comment (hovancik) [reply README.md:270]: This and other comments (about moving the *bsd related stuff to *bsd sections) I am not sure about: 
- on one hand it makes sense
- but then, kinda makes sense to group Books, Users Groups etc in one place. 

When looking for BSD groups, I would search for Groups section and not under the specific OS. But that's me.

**Pull request merged**